app/src/main/java/com/onboarding/wheatherapp/mvp/view/ActivityView.kt

```diff
@@ -1,11 +1,14 @@
 package com.onboarding.wheatherapp.mvp.view
 
 import android.app.Activity
+import android.content.Context
 import java.lang.ref.WeakReference
 
 open class ActivityView(activity: Activity) {
     private val activityRef: WeakReference<Activity> = WeakReference(activity)
-
     val activity: Activity?
         get() = activityRef.get()
+    val context: Context?
+        get() = activity
```

Comment:
you can put here activity.context

Reply:
According to what was discussed in the call, I will not modify it